### ISSUE TITLE
RNG Reporter Dark Grass Bugged + Sync Also Bugged

### ISSUE THREAD

**Issue by NickPlayeZ** [Bambo-Rambo/RNG-Guides#1]
Dark Grass doesnt work properly. 
Ive tried with 2 completely different seeds once in Virbank complex where I hit the shiny slot 7 multiple times before hitting the slot 5 (has 2 moving NPCs) that wasnt shiny despite the fact that it shouldve been according to rng reporter and the other time in the nature preserve. TID/SID 44000/15217
Seed 1: B606037187406E9E, Target Frame: 125, Wanted Slot: 5, Lead: Impish Sync
![1](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699972/c4c23455-cc4c-40f5-9f59-190f5d9ac8af)
![2](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699972/3716fdf6-0f9d-415e-8172-a380d7f8b345)

Seed 2: 3ABE62610C987CAE, Target Frame: 537, Wanted Slot: 2, Lead: Female Cute Charmer
![3](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699
![4](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699972/38362d83-c182-4862-b87f-8ecd957f8f43)
972/d
![5](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699972/e5ef042e-e245-4c30-8c83-709a8ac171f0)
b344f65-7def-47e2-9c2c-bd72b633fd4e)
![6 (I did use a female CC lead)](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699972/c87d5139-5589-41c6-94bb-8f5dae6aff54)

**Comment by NickPlayeZ** [comment]
Also Synchronize doesnt work properly either. This issue is not exclusive to Dark Grass.
Some examples I had recently: 

https://youtu.be/cZiZMW_lc38?t=1112
18:32
Seed: D1A57DD48AA857B8, Frame: 107, Method: Wild

https://youtu.be/3h9mOe2okFo?t=2688
44:53
Seed: 0ECB69D53C427CC1, Frame: 149, Method: Dark Grass

https://youtu.be/_Uh8F7E7IoQ?t=2711
45:11
Seed: 14AB7FD92C2BC077, Frame: 110, Method: Dark Grass


+a little inconsistency error: for Wild and Dark Grass methods at least you still have to stop 1 before your target frame and iirc Gift and Stationary have you be ON your target frame.

**Comment by Bambo-Rambo** [comment]
Synchronize and inaccurate frame indexation were infamous issues in the older versions but I am pretty sure they have been fixed in my fork.
I also tested Dark Grass again and I have been able to hit my targets without any issue:
``` Seed: 0E5ED65C5206BEA7 | Frames: 83/85 | TID: 65535 | SID: 57429```
![drif](https://github.com/Bambo-Rambo/RNG-Guides/assets/65967984/9922f546-9538-45d9-9789-b387a221b04e)
![skarm](https://github.com/Bambo-Rambo/RNG-Guides/assets/65967984/8d545f21-e137-4840-8c4c-28f30d9b8adc)

Memory Link could be the issue here but I don't have a save file with it used, and I don't have the time to do it now since I am leaving again tomorrow. If you could provide me with one, or test yourself in a save file without Memory Link used, we may be able to figure out the problem.

**Comment by NickPlayeZ** [comment]
Damn I didnt see this until now.
Well idk when youll be back but for whenever you return https://www.mediafire.com/file/x8ki34ioev8p06j/POKEMON_W2.sav/file
in the meantime I'll try and do some tests on a file without memory link soon. As for Dark Grass some seeds I used did work but I'd guesstimate about 66% of them didn't

**Comment by NickPlayeZ** [comment]
So I tried sync again with the first numel seed but this time on a no memory link savefile and this makes even less sense lol. I stopped 1 Frame before my target on frame 106 and ended up getting a 50/50 female slot 5 impish encounter (tried twice at reversal mountains outside for a level 33 drifblim and once on route 13 for a level 35 absol) which is nowhere to be seen. My lead was a modest umbreon with the synchronize ability.

Good news is I also stopped on the target frame and got the correct target with synchronized ability so that seems to work fine. So with memory link you still have to stop 1 before your target frame for wild but without memory link you stop ON your target frame
![grafik](https://github.com/Bambo-Rambo/RNG-Guides/assets/80699972/358e4110-315d-4af8-b0cf-31b50dd660c1)
